manifest.json

```
@@ -4879,6 +4879,29 @@
                 }
             }
         },
+        "me.catshark.KeepGlobalTransformToggle": {
+            "name": "KeepGlobalTransformToggle",
+            "description": "Adds a toggle to inspectors for the keepGlobalTransform used when moving objects in the inspector",
+            "category": "Inspectors",
+            "website": "https://github.com/CatSharkShin/KeepGlobalTransformToggle",
+            "sourceLocation": "https://github.com/CatSharkShin/KeepGlobalTransformToggle",
+            "authors": {
+                "CatSharkShin": {
+                    "url": "https://github.com/CatSharkShin"
+                }
+            },
+            "versions": {
+            "1.1": {
+                    "releaseUrl": "https://github.com/CatSharkShin/KeepGlobalTransformToggle/releases/tag/1.1",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/CatSharkShin/KeepGlobalTransformToggle/releases/tag/1.1",
```

This should be the URL of the actual DLL

```suggestion
                            "url": "https://github.com/CatSharkShin/KeepGlobalTransformToggle/releases/download/1.1/KeepGlobalTransformToggle.dll",
```